Core/HLE/HLE.h

```diff
@@ -166,7 +167,14 @@ T hleDoLog(LogTypes::LOG_TYPE t, LogTypes::LOG_LEVELS level, T res, const char *
 		va_end(args);
 	}
 
-	hleDoLogInternal(t, level, res, file, line, reportTag, retmask, reason, formatted_reason);
+	u64 fmtRes = res;
+	if (std::is_floating_point<T>::value) {
+		// We reinterpret as the bits for now, so we can have a common helper.
+		fmtRes = *(const u32 *)&res;
```

Comment:
Technically, should use memcpy for this, but we aren't compiling with strict aliasing so this will work.